Eatery/Controllers/Eateries/Campus/ScrollableViewController.swift

```diff
@@ -0,0 +1,283 @@
+//
+//  ScrollableViewController.swift
+//  Eatery
+//
+//  Created by Sergio Diaz on 12/9/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ScrollableViewController: UIViewController {
+
+    private let eatery: CampusEatery!
+    private let viewControllers: [CampusEateryExpandedMenuViewController]!  // The VCs for each menu section
+    private let originalViews: [UIView]!    // The views of the VCs for each menu section
+    private let allItems: [ExpandedMenu.Item]!  // All menu items combined into array and sorted
```

Comment:
These are good comments, can you move them above the variable declaration and with three slashes, like this?

```
/// This is a doc comment
private let var: Type
```

This way they'll show up when doing autocomplete, and also the comment can be extended without running into the 120 col limit